### PR TITLE
Finishing up throttle and dashboard stuff

### PR DESCRIPTION
**Pull request by byronwasti**

### CODE REVIEW

File: boards/BMS_master/read_cells.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ def display(self):
 
 
 if __name__ == '__main__':
-    can = CAN('/dev/cu.usbmodem14111')
```
Comment (aehoppe) [inline boards/BMS_master/read_cells.py]: What exactly is this change?

File: boards/BMS_master/main.c [hunk 2]
```diff
@@ -18,11 +18,15 @@ volatile uint8_t FLAGS = 0x00;
...
 #define AIRS_CLOSED         0b01000000
+#define IMD_TRIPPED         0b10000000
 
 #define PROG_LED_1 PB5
 #define PROG_LED_2 PB6
 #define PROG_LED_3 PC0
 
+#define IMD_SENSE_PIN      PC7
+#define IMD_SENSE_PIN_REG  PINC
+
```
Comment (aehoppe) [inline boards/BMS_master/main.c:29]: Why are there diffs in this file? Is this stuff new?

File: boards/Byron-Throttle/throttle.c [hunk 3]
```diff
@@ -0,0 +1,402 @@
+#include <avr/io.h>
```
Comment (aehoppe) [inline boards/Byron-Throttle/throttle.c:1]: What happened to the old throttle code?

File: boards/Dashboard/main.c [hunk 4]
```diff
@@ -220,16 +287,16 @@ int main(void) {
                         CAN_IDT_THROTTLE_L,
                         CAN_IDM_single);
     CAN_wait_on_receive(1,
-                        CAN_IDT_BMS_MASTER,
-                        CAN_IDT_BMS_MASTER_L,
+                        0x19,
```
Comment (aehoppe) [inline boards/Dashboard/main.c:301]: We should probably have all of this stuff up to date in the CAN api at some point

**Review by aehoppe — COMMENTED** [review]
I'd like you to answer the review questions here just so we can track what was going on, but otherwise it seems fine.

**Comment by segerpeter07** [comment]
Might need to double check can init on read_cells.py since it got kind of messed up...

**Pull request merged**